### DISCUSSION TITLE
QskTextInput

### DISCUSSION THREAD

**Discussion by hestrro** [uwerat/qskinny#461] [Q&A] [answered]
Can you add an interface to QskTextInput to control the selectionColor and selectedTextColor properties? Because by default they are all black, so the selected text cannot see its content.

**Comment by uwerat** [comment, marked as answer]
The subcontrols for these properties are: QskTextInput::PanelSelected, QskTextInput::TextSelected.

However initializing the colors has been forgotten for the Material3 skin. See: https://github.com/uwerat/qskinny/issues/466

You can override the settings from the skin in application code this way:

- input->setColor(  QskTextInput::PanelSelected, ... );
- input->setColor( QskTextInput::TextSelected, ... );
